### PR TITLE
refactor: cleanup `Toggle` component

### LINKED ISSUE #7091: [Task]: Cleanup `Toggle`

### PR DESCRIPTION
**Pull request by VmMad**
## Summary

Closes #7091 
Refactor props and styles of the Toggle component.

## Changelog

```
- Remove unnecessary classes prop
- Move all tailwind classes to the styles tag
```

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [x] Linux
    -   [ ] Windows
-   **Mobile**
    -   [ ] iOS
    -   [ ] Android

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [x] I have followed the contribution guidelines for this project
-   [x] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/shared/components/inputs/Toggle.svelte [hunk 1]
```diff
@@ -13,29 +19,46 @@
...
+<style lang="scss">
+    button {
+        @apply relative;
+        @apply block;
+        @apply w-10;
+        @apply shrink-0;
+        @apply h-6;
+        @apply rounded-full;
+        @apply border border-solid border-transparent dark:border-gray-700;
+        @apply cursor-pointer;
```
Comment (evavirseda) [inline packages/shared/components/inputs/Toggle.svelte:35]: do we need this? isn't a button's default `cursor-pointe`r?

**Review by evavirseda — CHANGES_REQUESTED** [review]
nice jean, just one small question :pray:

**Review by evavirseda — APPROVED** [review]
nice job :sunflower:

**Review by begonaalvarezd — APPROVED** [review]
💃🏼

**Pull request merged**